server/controllers/notification.js

```diff
@@ -0,0 +1,70 @@
+const Notification = require("../models/Notification");
+const User = require("../models/User");
+const asyncHandler = require("express-async-handler");
+
+// @route PUT /profile/edit
+// @desc edit user profile
+// @access Public
```

Comment:
We should either correct this piece of documentation or remove it. It's generally preferred to have at least basic documentation for all the endpoints 👍🏼

Reply:
Ok I added these types of notes to the endpoints here.